release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -409,6 +427,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+*Networking*
+
+* When running CNF tests on SNO and regular mode, the logic in place to understand if the cluster is ready is missing details. In particular, creating an SR-IOV network will not create a network attachment definition until at least one minute elapses. All the DPDK tests fail in cascade. Run the CNF tests skipping the DPDK feature when running against SNO in regular mode, with the -ginkgo.skip parameter. Run in discovery mode to execute tests against SNO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970409[*BZ#1970409*])
```

Comment:
Please expand SNO unless you feel confident that customers know what that is.

Maybe instead of "the logic in place to understand" can be "the tests failed to determine when the cluster is ready".  That might not be quite correct.  If you can make the sentence less wordy, please do.  

The use of "in cascade" might not be necessary and might not translate well.  I suggest removing it.

Please enclose `-ginko.skip` in backticks as suggested in the [quick reference](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference) section of the guidelines.